CoffeeChat/Models/Availability.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Availability.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 11/14/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+struct Availability: Codable {
```

Comment:
this model looks super similar to `DaySchedule` aside from the type of the `times` array -- what's the difference between the two/could they be consolidated?